content/md/en/docs/reference/substrate-connect.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Substrate Connect
+description: Run Wasm Light Clients of any Substrate based chain directly in your browser
+keywords:
+---
+Substrate is a modular framework for creating use-case optimized blockchains at a low cost, by composing custom or pre-built components. Substrate is the backbone that powers Polkadot, a next-generation, heterogeneous, multi-chain network, and its ecosystem.
+
+## Wasm Light Clients
+
+Substrate-connect turns a browser into a node on a network allowing end-users of Web3 apps to interact with blockchains directly - without connecting to third-party remote nodes and other servers. Removing intermediary servers between network and its users improves security, simplifies infrastructure of a network and lowers its maintenance costs. Decentralized in-browser light clients are a unique feature of substrate-based networks.
+
+[What is a light client and why you should care? by Thibault Sardan](https://www.parity.io/what-is-a-light-client/)
+
+## Well known Networks
+
+* [Polkadot](https://polkadot.network/) (`polkadot`)
+* [Kusama](https://kusama.network/) (`ksmc33`)
+* [Westend](https://wiki.polkadot.network/docs/en/maintain-networks#westend-test-network) (`westend2`)
+* [Rococo](https://polkadot.network/rococo-v1-a-holiday-gift-to-the-polkadot-community/) (`rococo_v2_2`)
+
+[WellKnownChain enumeration docs](https://paritytech.github.io/substrate-connect/api/enums/connect_src.WellKnownChain.html)
+
+You have to provide a well-known chain name (`polkadot` for Polkadot, `ksmcc3` for Kusama, `westend2` for Westend and `rococo_v2_2` for Rococo). Note that these are the "real" names of the chains rather than the names they are more commonly known as (such as Kusama or Rococo). For example, "ksmcc3" is the name of Kusama. This is important for chains which have been hard forked. For example, "rococo_v2" and "rococo_v2_2" are two different chains.
+
+## Usage with PolkadotJS API (PolkadotJS provider)
+
+For using `substrate-connect` with the existing polkadotJS api, there is an rpc-provider that has `substrate-connect` already integrated. You can use that by installing in your app, the following packages:
+#### **@polkadot/rpc-provider**
+This package contains the `substrate-connect` rpc provider that integrates and uses substrate-connect:
+```bash
+yarn add @polkadot/rpc-provider
+```
+*or*
```

Comment:
Generally it is better to pick a preferred path.

Reply:
We cannot really - it is the developers decisions if he will use `npm` or `yarn`. We just provide both alternatives. IMO choosing one path would be like, not supporting the alternative (which we do)